nss-cohort-40/bangazon-ecommerce-client-upperclasstwits
# NavBar and BangazonBuilder

# Description

Completed NavBar and BangazonBuilder.

Fixes # (issue)
## Type of change

- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

Run npm start in terminal. Home page should load and display navbar links. LINKS DONT WORK YET.

# Checklist:
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
